composer.json

```diff
@@ -34,6 +34,11 @@
             "Exporter": "lib/"
         }
     },
+    "autoload-dev": {
+        "psr-0": {
```

Comment:
@core23 fixed… sort of… I think I'll make a separate PR and rebase on it if this turns out to rot.